app/views/my_listings/destroy.html.erb

```diff
@@ -0,0 +1,4 @@
+<div>
```

Comment:
I'm seeing a few of these default files. Are you going to tackle those in another issue/pr?